packages/data/src/batcher.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ReadStream } from 'fs';
+import { Connection, Logger, Messages, SfdxError } from '@salesforce/core';
+import parse = require('csv-parse');
+import { Batch, BatchInfo, BatchResultInfo, JobInfo } from 'jsforce';
+import { UX } from '@salesforce/command';
```

Comment:
I don't think this library should use anything from `@salesforce/command`.  In fact, if it's a dependency listed in package.json we should remove it.

Also, I think we should always use `fs` from `@salesforce/core` since it protects us against some problems we've run into with the native fs lib.